ArchiSteamFarm/Steam/Interaction/Commands.cs

```diff
@@ -46,6 +46,7 @@ namespace ArchiSteamFarm.Steam.Interaction;
 
 public sealed class Commands {
 	private const ushort SteamTypingStatusDelay = 10 * 1000; // Steam client broadcasts typing status each 10 seconds
+	private const byte EAssetRarities = 10; // Currently, we recognize 10 EAssetRarity values
```

Comment:
It seems this constant is used only for optimization - in this case I'd prefer to not have it at all, it doesn't hurt us to extract extra entries from the input, if any.